container-chains/runtime-templates/frontier/src/lib.rs

```diff
@@ -572,9 +573,20 @@ impl pallet_utility::Config for Runtime {
 }
 
 /// The type used to represent the kinds of proxying allowed.
-#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
 #[derive(
-    Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug, MaxEncodedLen, TypeInfo,
+    Copy,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Encode,
+    Decode,
+    Debug,
+    MaxEncodedLen,
+    TypeInfo,
+    Serialize,
+    Deserialize,
```

Comment:
Actually we don't need serialize here, not sure why did we add that. For the other derives I don't care if they are unused, especially the ones from the standard library, but serde is an external crate so it would be nice to only use it when needed
```suggestion
```